dss/storage/checkout/__init__.py

```diff
@@ -57,7 +59,7 @@ def touch_test_file(replica: Replica, dst_bucket: str) -> bool:
     :param replica: the replica to execute the checkout in.
     :return: True if able to write, if not raise DestinationBucketNotWritableError.
     """
-    test_object = "touch.txt"
+    test_object = f"touch/{''.join(random.choices(hexdigits, k=4))}.txt"
```

Comment:
This will eventually create over two hundred thousand files:
```
In [11]: len(string.hexdigits)**4
Out[11]: 234256
```

Reply:
the files will be cleaned up by the buckets data life cycle setting before that size is reached. If that size is reached it will only consume a few MB of storage. Also you can set a specific data life cycle with files with the touch* as the prefix to clean them up soon after they are created. Either way I don't think this is an issue.